lib/jsdom/living/helpers/document-base-url.js

```diff
@@ -28,7 +28,7 @@ exports.fallbackBaseURL = document => {
     return exports.documentBaseURL(document._defaultView._parent._document);
   }
 
-  return document._URL;
+  return whatwgURL.parseURL(document.URL);
```

Comment:
Is this change necessary?

Reply:
yes it is required to pass the tests for the base element.

i debugged this and because of my change in the iframe implementation the code now reaches  `return exports.documentBaseURL(document._defaultView._parent._document);` on line 28 and as far as i can tell we only have the public api `document.URL` to work with if we access the parent document this way. thats why i cannot access `document._URL` anymore and i think this code cannot work in other scenarios aswell because of this.

if you know a better way to do this then i would be interested to see it :)